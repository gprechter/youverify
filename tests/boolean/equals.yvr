t, f: BOOL
a, b, c, d: BOOL
t = true
f = false
a = 1 == 1
b = 1 == 2
c = 2 == 1
d = 2 == 2