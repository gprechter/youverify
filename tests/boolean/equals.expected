{
    "t": true,
    "f": false,
    "a": true,
    "b": false,
    "c": false,
    "d": true
}